pandas/conftest.py

```diff
@@ -103,6 +120,18 @@ def all_arithmetic_operators(request):
     return request.param
 
 
+_cython_table = list(pd.core.base.SelectionMixin._cython_table.items())
```

Comment:
why not always sort them?

Reply:
for Python >= 3.6 dicts are already sorted and I think maintaining dict order makes the most sense.

The sorting is only because xdist fails for Python 3.5, so predefined order is necessary for Python < 3.6 to satisfy xdist.

Reply:
i understand, but this is for testing where we want determinism. Having alphabeting ordering is good.

Reply:
It is deterministic for python 3.6 also, as per the ordered dicts. My thinking is that the if not if ``if`` part can more easily be removed when support for python 3.5 is dropped and ``PY36`` is removed from compat...

However, this is obviously not a huge issue for me, I'll can change it.